massa-graph/src/block_graph.rs

```diff
@@ -451,8 +443,9 @@ impl BlockGraph {
                     addresses_to_operations: Map::with_capacity_and_hasher(0, BuildMap::default()),
                     roll_updates: RollUpdates::default(), // no roll updates in genesis blocks
                     production_events: vec![],
-                    block,
+                    block_id,
```

Comment:
As you only pass the id you shouldn't store the block just above ?

Reply:
Correct, thanks